Gemfile

```diff
@@ -38,3 +38,7 @@ else
   end
 end
 # END ENGINE_CART BLOCK
+
+unless File.exist?(file)
+  eval_gemfile File.expand_path('spec/test_app_templates/Gemfile.extra', File.dirname(__FILE__))
```

Comment:
@jcoyne sure. Will `bundle` get re-run or do I have to call that somehow? BTW, I was following the same pattern in Hyrax, but maybe this one here is more current?